src/components/Layout/Navbar/Navbar.tsx

```diff
@@ -84,6 +85,11 @@ const Navbar: React.FC = () => {
                 <Text className={styles.antTypography}>Join Us</Text>
               </a>
             </div>
+            {user !== null ? (
+              <Link to="/mentor-registration">
+                <Button className={styles.loginButton}>Become a Mentor</Button>
+              </Link>
+            ) : null}
```

Comment:
This should be visible without login. when they try to sign up we check whether they are logged in or not.

Reply:
sending in